docs/notebooks/Training_with_custom_hyperparameters.ipynb

```diff
@@ -340,7 +340,7 @@
         "id": "zDbIRjlw_fOF"
       },
       "source": [
-        "The line below assumes your custom training profile is called `multi_instance.json` and that you've copied it into the `sleap-custom-training` directory in your Google Drive, along side your training package called `dataset.pkg.slp`. Edit the filenames in the command to match your files."
+        "The line below assumes your custom training profile is called `multi_instance.json` and that you've copied it into the `sleap-custom-training` directory in your Google Drive, along side your .slp file from training job package called `dataset.pkg.slp`. Edit the filenames in the command to match your files."
```

Comment:
Why aren't we using the zip file-based workflow here?

It's being used in the other notebook, but it's actually _more_ relevant here.

The notebook with defaults params doesn't even use the other files in the zip file, but here we actually use the json files that specify these training configs.

Reply:
I didn't work on this notebook in this PR - let me update it as well now